@here/harp-mapview/lib/MapView.ts

```diff
@@ -1047,6 +1052,16 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_tileGeometryManager;
     }
 
+    get enableMixedLod(): boolean {
+        return this.m_enableMixedLod;
+    }
+
+    set enableMixedLod(enableMixedLod: boolean) {
+        this.m_enableMixedLod = enableMixedLod;
+        this.m_visibleTiles = this.createVisibleTileSet();
```

Comment:
Shouldn't you clear the tile cache? Maybe it's fine to not clear it b/c some of the tiles might be reused.

Reply:
And maybe clearing the text elements render states as well?